connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/WorkerCoordinator.java

```diff
@@ -267,6 +267,18 @@ public String memberId() {
         return JoinGroupRequest.UNKNOWN_MEMBER_ID;
     }
 
+    @Override
+    protected void handlePollTimeoutExpiry() {
+        log.warn("worker poll timeout has expired. This means the time between subsequent calls to poll() " +
+            "in DistributedHerder tick() method was longer than the configured rebalance.timeout.ms. " +
+            "If you see this happening consistently, then it can be addressed by either adding more workers " +
+            "to the connect cluster or by increasing the rebalance.timeout.ms configuration value. Please note that " +
+            "rebalance.timeout.ms also controls the maximum allowed time for each worker to join the group once a " +
+            "rebalance has begun so the set value should not be very high");
```

Comment:
@gharris1727 @mimaison , could you help review these log output since you're the connect experts. Thanks.

Reply:
Mickael/Greg could you please review the above message to see if it makes sense for connect? Thanks!